app/controllers/application_controller.rb

```diff
@@ -85,6 +85,7 @@ def authenticate_token
     authenticate_with_http_token do |token, _options|
       @session = Session.find(token)
       return false if @session.nil?
+      Rails.logger.info("Logged in user with id #{@session.uuid}")
```

Comment:
Sorry, so.. at past jobs we've included some user-identifiable trait in the log tags themselves. Most (all?) users will have a google analytics cookie set, and any authenticated user with have an authorization token in the request header. In addition to this log message, which would appear once per request, we could add either the GA or Auth token (or whatever else) as tags (like the uuid of the request) to each _line_ of the log for the request. That would make it easier to search for all the log activity for a given user once the trait was identified.

My thinking with 'part' and 'subset' would be that if either the full authorization token or google analytics cookie is too identifiable, we could snip out a smaller substring of either of them.